### PR TITLE
installed noty and did ng-click to post bookmarks

### PR DESCRIPTION
**Pull request by cchchoi1986**

### CODE REVIEW

**Comment by h8rry** [comment]
You need to rebase first. It's not auto-mergable

File: app/assets/javascripts/angular/controllers/pages/home_controller.js.erb [hunk 1]
```diff
@@ -1,101 +1,385 @@
...
-   "http://images.media-allrecipes.com/userphotos/140x140/00/83/68/836815.jpg",
-   "http://images.media-allrecipes.com/userphotos/140x140/00/00/99/9997.jpg",
-   "http://images.media-allrecipes.com/userphotos/140x140/00/28/96/289632.jpg",
-   "http://images.media-allrecipes.com/userphotos/140x140/00/96/12/961261.jpg",
-   "http://images.media-allrecipes.com/userphotos/140x140/01/83/64/1836491.jpg",
-   "http://images.media-allrecipes.com/userphotos/140x140/00/08/49/84924.jpg",
-   "http://images.media-allrecipes.com/userphotos/140x140/01/00/06/1000669.jpg",
-   "http://images.media-allrecipes.com/userphotos/140x140/00/52/23/522354.jpg",
-   "http://images.media-allrecipes.com/userphotos/140x140/00/80/71/807159.jpg"
+  "http://images.media-allrecipes.com/userphotos/140x140/00/43/73/437365.jpg",
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/pages/home_controller.js.erb]: Just store `/00/43/73/437365.jpg`, not the whole url.

File: app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb [hunk 2]
```diff
@@ -1,8 +1,18 @@
 app.controller('RecipeCtrl', ['$scope', '$http', '$routeParams', function($scope, $http, $routeParams){
 
+  $scope.defaultImg = "<%= asset_path('recipedb-default-img.png') %>"
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb:3]: semi-colon

File: app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+
+  $scope.sendNoty = function (notyType, notyMessage) {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb:5]: no space after `function`

File: app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
+    noty({
+      text: notyMessage,
+      type: notyType
+    })
+  }
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb:10]: semi-colon

File: app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb [hunk 5]
```diff
@@ -93,9 +94,18 @@ app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', 'TagServi
   // add to collection button
   $scope.addCollection = function(recipeID) {
     console.log(recipeID);
-    $http.post('/user/' + UserServices.userID + '/bookmarks/', recipeID).success(function(data, status){
-      // console.log('data');
+    // $scope.liked = !$scope.liked;
+    $http.post('/bookmarks/'+recipeID).success(function(data, status){
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb:98]: Work in progress? I thought it's done

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -3,7 +3,19 @@ def any
...
+    bookmark.recipe_id = params[:recipe_id]
+    if bookmark.save
+      render json: {success: true}
+    else
+      render json: {success: false}
+    end
+  end
+
+  def remove_bookmark
+    bookmark = current_user.user_bookmarks.where(recipe_id: params[:recipe_id])
```
Comment (h8rry) [inline app/controllers/users_controller.rb]: If you are only destroy one record, you should use `find`

File: app/assets/javascripts/angular/controllers/pages/home_controller.js.erb (hunk 1 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/pages/home_controller.js.erb]: ok will do later

File: app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb (hunk 3 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb:5]: ok changed

File: app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb (hunk 2 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb:3]: changed

File: app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb (hunk 4 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/pages/recipe_controller.js.erb:10]: changed

File: app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb (hunk 5 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb:98]: yea its done now

File: app/controllers/users_controller.rb (hunk 6 above)
Comment (cchchoi1986) [inline app/controllers/users_controller.rb]: done

File: app/assets/javascripts/angular/controllers/pages/home_controller.js.erb (hunk 1 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/pages/home_controller.js.erb]: done

File: app/controllers/users_controller.rb (hunk 6 above)
Comment (cchchoi1986) [inline app/controllers/users_controller.rb]: find_by?

**Pull request merged**